#!/usr/bin/env python

PACKAGE = "pointcloud_to_rangeimage"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_range", double_t, 0, "the sensor minimum range", 0, 0, 2)
gen.add("max_range", double_t, 0, "the sensor maximum range", 200, 0, 200) #default 200
gen.add("laser_frame", bool_t, 0, "the range image sensor frame (default laser)", True)

exit(gen.generate(PACKAGE, "pointcloud_to_rangeimage", "PointCloudToRangeImageReconfigure"))
